From 0d1445def23f4020e91506c9cc3af711eaa1b3b3 Mon Sep 17 00:00:00 2001
From: Nishanth Menon <nm@ti.com>
Date: Thu, 19 Sep 2013 14:11:36 -0500
Subject: [PATCH 58/58] ARM: dts: omap5-uevm: mark TWL6037 as
 system-power-controller

This allows the palmas pm_power_off to kick in on power off command
and switch off the board.

Signed-off-by: Nishanth Menon <nm@ti.com>
Signed-off-by: Benoit Cousson <bcousson@baylibre.com>
---
 arch/arm/boot/dts/omap5-uevm.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/omap5-uevm.dts b/arch/arm/boot/dts/omap5-uevm.dts
index 748f6bf..d784b3a 100644
--- a/arch/arm/boot/dts/omap5-uevm.dts
+++ b/arch/arm/boot/dts/omap5-uevm.dts
@@ -249,6 +249,7 @@
 		reg = <0x48>;
 		interrupt-controller;
 		#interrupt-cells = <2>;
+		ti,system-power-controller;
 
 		extcon_usb3: palmas_usb {
 			compatible = "ti,palmas-usb-vid";
-- 
1.8.4.rc3

